src/docs/product/data-management-settings/scrubbing/server-side-scrubbing/index.mdx

```diff
@@ -68,6 +68,36 @@ If you’ve accidentally sent sensitive data to the server, you likely don't to
 - While you cannot delete a single event, you can delete the issue the event is in by clicking the trash icon on the **Issue Details** page.
 - If a large number of unique events - representing multiple issues - contain the data, you may need to delete and re-create the project to effectively cleanse the system.
 
+## Safe Fields
+
+Safe Fields are used to exclude fields and data from scrubbing. For example this event payload:
+
+```json
+{
+  "user": {
+    "id": "4111111111111111"
+  },
+  "extra": {
+    "id": "sensitive",
+    "sys.argv": ["script.py", "password"],
+    "credentials": { "username": "jane", "password": "p4ssw0rd!" }
+  }
+}
+```
+
+With scrubbing enabled and `id` added as a sensitive field, the `user.id` field will be scrubbed
+(it matches a credit card), the `password` string in `extra.'sys.argv'` as well as the entire `credentials` object.
```

Comment:
```suggestion
When scrubbing is enabled and `id` added as a sensitive field, the `password` string in `extra.'sys.argv'`, the `user.id` field (because it matches a credit card), as well as the entire `credentials` object will be scrubbed.
```